test/integration/targets/apt/tasks/apt_deb_depend.yml

```
@@ -0,0 +1,41 @@
+- block:
+  - name: Clean up before running tests
+    apt:
+      name:
+        - packageone
+        - packagetwo
+      state: absent
+
+  - name: Install packageone from deb URL and make sure it installs depend
+    apt:
+      deb: https://ci-files.testing.ansible.com/test/integration/targets/apt/packageone_1.0_all.deb
+      state: present
+    register: packageone_installed
+
+  - name: Check if packagetwo is installed as part of packageone installation
+    shell: dpkg -s packagetwo
+    register: packagetwo_installed
+
+  - name: Make sure packagetwo is installed
+    assert:
+      that:
+        - packagetwo_installed.rc == 0
+
+  - name: Check if the packagetwo is marked as auto
+    shell: apt-mark showauto
+    register: packagetwo_auto
```

Check the status of only packageone and packagetwo:

```suggestion
  - name: Check if packageone and packagetwo are marked as auto
    shell: apt-mark showauto packageone packagetwo
    register: auto_installed
```